client/my-sites/site-settings/form-writing.jsx

```diff
@@ -62,26 +62,32 @@ class SiteSettingsFormWriting extends Component {
 			uniqueEventTracker,
 			fields,
 			handleSelect,
+			handleSubmitForm,
 			handleToggle,
 			handleAutosavingToggle,
 			handleAutosavingRadio,
+			isJetpackSite: _isJetpackSite,
 			isRequestingSettings,
 			isSavingSettings,
+			jetpackMasterbarSupported,
+			jetpackSettingsUISupported,
 			onChangeField,
 			setFieldValue,
 			siteId,
 			translate,
 			updateFields,
 		} = this.props;
 
+		const jetpackSettingsUI = _isJetpackSite && jetpackSettingsUISupported;
```

Comment:
Really like combining the two makes the code easier to read. Wondering if there could be a more descriptive name though. Not that I can think of one off the top of my head 🤔